test_vector_handlers/src/awses_test_vectors/manifests/master_key.py

```diff
@@ -50,11 +50,17 @@
     "rsa/pkcs1": WrappingAlgorithm.RSA_PKCS1,
     "rsa/oaep-mgf1/sha1": WrappingAlgorithm.RSA_OAEP_SHA1_MGF1,
     "rsa/oaep-mgf1/sha256": WrappingAlgorithm.RSA_OAEP_SHA256_MGF1,
-    # \/ not yet implemented \/
-    # 'rsa/oaep-mgf1/sha384': WrappingAlgorithm.RSA_OAEP_SHA384_MGF1,
-    # 'rsa/oaep-mgf1/sha512': WrappingAlgorithm.RSA_OAEP_SHA512_MGF1,
 }
-_NOT_YET_IMPLEMENTED = {"rsa/oaep-mgf1/sha384", "rsa/oaep-mgf1/sha512"}
+try:
+    _RAW_WRAPPING_KEY_ALGORITHMS.update(
+        {
+            "rsa/oaep-mgf1/sha384": WrappingAlgorithm.RSA_OAEP_SHA384_MGF1,
+            "rsa/oaep-mgf1/sha512": WrappingAlgorithm.RSA_OAEP_SHA512_MGF1,
```

Comment:
Currently the AWS Encryption SDK for C supports only RSA_OAEP_SHA1_MGF1 and RSA_OAEP_SHA256_MGF1. I can skip this in the test vector framework, but I was wondering if we need to make this clear in our documentation, as I am working with June on this.